storage/migrations/12_analysis_node_height.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+
+-- Tracks the current (consensus) height of the node. TODO: expand to paratime heights.
+CREATE TABLE chain.node_heights
```

Comment:
`chain.latest_node_heights` to make it clearer that this only tracks the latest node heights, not all heights.